src/ui/Analytics/Analytics.ts

```diff
@@ -371,6 +373,42 @@ export class Analytics extends Component {
     this.client.setOriginContext(originContext);
   }
 
+  /**
+   * Re-enables analytics if they were previously disabled.
+   */
+  public enable() {
+    if (!this.disabled) {
+      console.warn('Analytics are already enabled.');
+      return;
+    }
+    super.enable();
+    this.initializeAnalyticsClient();
+    this.resolveQueryController().historyStore = new HistoryStore();
+  }
+
+  /**
+   * Removes the analytics tracking cookie and clears the actions history.
+   */
+  public clearLocalData() {
+    this.client.endpoint.forgetVisitorId();
```

Comment:
Since this is only available on a "live" analytics client, I think we'd need a check here to verify that the `client` is not `Noop` or this would crash here.

So I guess this function should also first verify that the component is not currently disabled.

Reply:
Oh damn I missed that! I resolved it, now.